script/install_jessie.sh

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env bash
+
+# Uncomment the following three line have to be uncommented if Updation of Arduino alone is done seperately
+#sudo apt-get update
+#sudo apt-get upgrade
+#sudo apt-get dist-upgrade
+
+
+
+###*******Install.sh Starts+**********###
+
+
+echo "  _____            _                                ";
+echo " |  __ \          | |                               ";
+echo " | |  | | _____  _| |_ ___ _ __                     ";
+echo " | |  | |/ _ \ \/ / __/ _ \ '__|                    ";
+echo " | |__| |  __/>  <| ||  __/ |                       ";
+echo " |_____/ \___/_/\_\\__\___|_| _        _            ";
+echo " |_   _|         | |         | |      (_)           ";
+echo "   | |  _ __   __| |_   _ ___| |_ _ __ _  ___  ___  ";
+echo "   | | | '_ \ / _\` | | | / __| __| '__| |/ _ \/ __|";
+echo "  _| |_| | | | (_| | |_| \__ \ |_| |  | |  __/\__ \ ";
+echo " |_____|_| |_|\__,_|\__,_|___/\__|_|  |_|\___||___/ ";
+echo "                                                    ";
+echo "                                                    ";
+echo " "
+printf "Welcome to Arduberry Installer.\nPlease ensure internet connectivity before running this script.\n
+NOTE: Raspberry Pi wil reboot after completion."
+printf "Special thanks to Joe Sanford at Tufts University.  This script was derived from his work.  Thank you Joe!"
+printf " "
+echo "Must be running as Root user"
```

Comment:
Do a test and quit if not root 
if [[ -f /home/pi/quiet_mode ]]
then
    quiet_mode=1
else
    quiet_mode=0
fi

Then if quiet_mode, you know you're being run from DI Update. No need to do an internet connection, no need to display big dexter banner